app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split(" ")
+    if command_list[1] != command_list[2] and command_list[0] == "cp":
```

Comment:
you can unpack these variables in the easiest way, also use the more descriptive names
command, old_file, new_file = command.split()

Reply:
Done :)